drivers/SmartThings/zigbee-thermostat/profiles/thermostat-temperature-temperaturealarm.yml

```diff
@@ -10,8 +10,21 @@ components:
     version: 1
   - id: thermostatOperatingState
     version: 1
+    configuration:
+      values:
+        - key: "thermostatOperatingState.value"
+          enabledValues:
+            - Heating
+            - Idle
   - id: temperatureAlarm
     version: 1
+    configuration:
+      values:
+        - key: "thermostatOperatingState.value"
+          enabledValues:
+            - Normal
+            - Overheating
+            - Freezing
```

Comment:
```suggestion
        - key: "temperatureAlarm.value"
          enabledValues:
            - cleared
            - freeze
            - heat
```

Reply:
Looks like there are a number of similar problems with `temperatureAlarm` in other profiles. @greens it may make more sense for you or someone from your team to fix those up rather than asking @iplantemn to figure out what is correct.